src/components/MainPageMovieContainer.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react'
+import { Card, Container, Image, Grid } from 'semantic-ui-react'
+import axios from 'axios'
+import _ from 'lodash'
```

Comment:
What is this?

Reply:
we are using lodash to round down our average rating number on the top 10 global movies to 1 decimal point